SupremacyCore/Combat/AutomatedCombatEngine.cs

```diff
@@ -788,65 +920,65 @@ protected override void ResolveCombatRoundCore()
                     {
-                        if(commandShipModifierAccuracy < 0) // only if modifyer is negative
+                        if (commandShipModifierAccuracy < 0) // only if modifyer is negative
                         {
-                            sourceAccuracy = sourceAccuracy + commandShipModifierAccuracy; 
+                            sourceAccuracy = sourceAccuracy + commandShipModifierAccuracy;
                         }
                     }
 
                     // update x 23 july 2019: added experience to change accuracy and damage control: Checking Target experience in Attacking Loop
-                        string TargetShipExperience = "Unknown";
-                            TargetShipExperience = currentTarget.Item1.Source.ExperienceRankString;
-                            switch (TargetShipExperience)
-                            {
-
-                                case "Unknown":
-                                    {
-                                        //sourceAccuracy = 0.75;
-                                        targetDamageControl = 0.55;
-                                        break;
-                                    }
-                                case "Green":
-                                    {
-                                        //sourceAccuracy = 0.45;
-                                        targetDamageControl = 0.40;
-                                        break;
-                                    }
-                                case "Regular":
-                                    {
-                                        //sourceAccuracy = 0.50;
-                                        targetDamageControl = 0.45;
-                                        break;
-                                    }
-                                case "Veteran":
-                                    {
-                                        //sourceAccuracy = 0.55;
-                                        targetDamageControl = 0.50;
-                                        break;
-                                    }
-                                case "Elite":
-                                    {
-                                        //sourceAccuracy = 0.65;
-                                        targetDamageControl = 0.57;
-                                        break;
-                                    }
-                                case "Legendary":
-                                    {
-                                        //sourceAccuracy = 0.75;
-                                        targetDamageControl = 0.65;
-                                        break;
-                                    }
+                    string TargetShipExperience = "Unknown";
+                    TargetShipExperience = currentTarget.Item1.Source.ExperienceRankString;
```

Comment:
There is an enum for experience rank, which would make the converting it to string to compare it in code a pointless excercise